data/processing/input_latency.html

```diff
@@ -17,7 +17,7 @@
 
 tr.exportTo('pi.m', function() {
 
-  var LATENCY_EVENT = 'InputLatency::GestureScrollUpdate';  // Latency event to be analyzed
+  var LATENCY_EVENT = 'GestureScrollUpdate';                // Latency event to be analyzed
```

Comment:
It's probably possible but no other mappers do this. I skipped it because it wasn't immediately clear how to do it and I would have to do it two different ways since the mapper can be run locally or remotely.